src/pages/BO/dashboard/index.ts

```diff
@@ -0,0 +1,11 @@
+import semver from "semver";
+import BOBasePage from "../BOBasePage";
+
+let dashboardPage: BOBasePage;
+if (semver.gte(process.env.PS_VERSION as string, '8.0.0')) {
+    dashboardPage = require("@versions/8.0.0/pages/BO/dashboard");
```

Comment:
```suggestion
    dashboardPage = await import("@versions/8.0.0/pages/BO/dashboard");
```

For Typescript

Reply:
`Top-level 'await' expressions are only allowed when the 'module' option is set to 'es2022', 'esnext', 'system', 'node16', or 'nodenext', and the 'target' option is set to 'es2017' or higher.`